scripts/install/install_macos.sh

```diff
@@ -360,106 +415,104 @@ set_opamp_secret_key()
   fi
 }
 
-set_formula_name()
+# latest_version gets the tag of the latest release.
+# It's also super hacky; It assumes that the output is formatted prettily.
+# Ideally we'd use jq, but we want to avoid external dependencies
+latest_version()
 {
-  # If version flag is not set use the COLLECTOR_VERSION
-  if [ -z "$version" ] ; then
-    version=$COLLECTOR_VERSION
-  fi
-
-  # If version is set append it to formula name
-  if [ -n "$version" ] ; then
-    FORMULA_NAME="$FORMULA_NAME@$version"
-  fi
+  curl -sSL -H"Accept: application/vnd.github.v3+json" https://api.github.com/repos/observIQ/observiq-otel-collector/releases/latest | \
+    grep "\"tag_name\"" | \
+    sed -E 's/ *"tag_name": "v([0-9]+\.[0-9]+\.[0-9+])",/\1/'
 }
 
-check_install_exists()
+# This will install the package by downloading & unpacking the tarball into the install directory
+install_package()
 {
-  set +e
-  brew list "$FORMULA_NAME" > /dev/null 2>&1
-  install_exists=$?
-  set -e
-}
+  banner "Installing observIQ OpenTelemetry Collector"
+  increase_indent 
 
-find_existing_formula_name()
-{
-  # This can return multiversion we are currently allowing this to go into a failure state.
-  # This can be fixed by the user by specifiying the version via the -v flag.
-  # This will be resolved in the future by either supporting multi versions correctly or only allowing a single version install.
-  set +e
-  found_name=$(brew list --full-name | grep "observiq/observiq-otel-collector/observiq-otel-collector")
-  set -e
-
-  if [ -n "$found_name" ]; then
-    FORMULA_NAME=$found_name
-  fi
-}
+  # Remove temporary directory, if it exists
+  rm -rf "$TMP_DIR"
+  mkdir -p "$TMP_DIR"
```

Comment:
This isn't needed as `-p` creates all missing directories so we can just call `mkdir -p "$TMP_DIR/artifacts"`